src/main/webapp/app/entities/accessibility-for-disabilities/accessibility-for-disabilities.tsx

```diff
@@ -46,7 +51,37 @@ export class AccessibilityForDisabilities extends React.Component<IAccessibility
   }
 
   componentDidMount() {
-    this.getEntities();
+    if (!_.isEqual(this.props.location.search, '')) {
+      const parsed = queryString.parse(this.props.location.search);
```

Comment:
You could move this to separate function and call it in these 3 places: https://github.com/benetech/ServiceNet/pull/635/files#diff-02bac1b50216c64d8296a4618de767a9R55
https://github.com/benetech/ServiceNet/pull/635/files#diff-02bac1b50216c64d8296a4618de767a9R70
https://github.com/benetech/ServiceNet/pull/635/files#diff-02bac1b50216c64d8296a4618de767a9R78

Reply:
I don't like the naming of it, i'd prefer fetchPageData something similar.